.github/ISSUE_TEMPLATE/2-localization-fix-requests

```diff
@@ -0,0 +1,103 @@
+name: Localization Fix Requests
+description: File a request to fix translated text in the Taxonomy Tree
+title: "[Localization Fix]: "
+assignees: aalexandreia,MRodriguezRod
+labels:
+  - "localization-fix-requests"
+body:
+  - type: dropdown
+    id: localization_issue_type
+    attributes:
+      label: Are you reporting a Mixed Language Experience (MLX) or a Translation Quality issue?
+      multiple: false
+      options:
+        - Translation Quality issue
+        - MLX (English text in another language)
+  - type: dropdown
+    id: target_language
+    attributes:
+      label: Target Language(s)
```

Comment:
Do we have stats on language use? Should we come up with some heuristic on what language gets listed vs "othered"?

Reply:
@aalexandreia or @MRodriguezRod can confirm, but I believe this matches the languages we support in our customer accounts (excluding beta CAN EN we don't support in the public repo right now):
![image](https://github.com/user-attachments/assets/0205df20-6586-4805-9c43-4ffd9f357443)

Reply:
@ricardotejedorsanz I guess my mind went to dropping `other` if we have this as exhaustive. But suppose it doesn't hurt to having an escape hatch.

Reply:
What would be the case scenario where "other" would be selected, given that we have all lang and allow for multiple selections?

Reply:
In that case we can probably drop `other`. This isn't a mandatory field, so users can always leave it blank if a language isn't listed (e.g. suggesting we support another lang). We can always go back to adding `other` later.